stripe-ui-core/src/main/java/com/stripe/android/uicore/image/ImageLruMemoryCache.kt

```diff
@@ -0,0 +1,41 @@
+package com.stripe.android.uicore.image
+
+import android.graphics.Bitmap
+import android.util.LruCache
+import androidx.annotation.RestrictTo
+
+@RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
+class ImageLruMemoryCache {
```

Comment:
Can you add a Javadoc class description for all classes?

Reply:
https://github.com/stripe/stripe-android/pull/5307/commits/594946c89a7d67a5f05670e6425bf60687afbec5